src/modules/vendor/components/programs/ProfileInfo.vue

```diff
@@ -16,7 +16,16 @@
       <p class="text-weight-bold">Modules</p>
       <div v-for="(module, index) of program.modules" :key="index">
         <q-card flat class="module-card">
-          <q-card-section>{{module.title}}</q-card-section>
+          <q-card-section class="cursor-pointer" @click.native="showActivities(module._id)">
+            {{module.title}}
+          </q-card-section>
+          <div class="beige-background activities" v-if="isActivitiesShown[module._id]">
+            <div v-for="(activity, index) of module.activities" :key="index">
+              <q-card><q-card-section>{{activity.title}}</q-card-section></q-card>
+            </div>
```

Comment:
tu peux mettre le v-for direct sur le q-card non ?

Reply:
Done